src/open_inwoner/templates/pages/kvk/branches.html

```diff
@@ -2,36 +2,106 @@
+            <form method="POST" id="eherkenning-branch-form" class="form">
+                {% csrf_token %}
+                {% if form.branch_number.errors %}
+                    {% errors errors=form.branch_number.errors %}
+                {% endif %}
+
+                <ul class="choice-list choice-list--single">
+
+                {# if COMPANY shows NEVENVESTIGING and HOOFDVESTIGING in one list #}
+                <p><strong>{% trans "Selecteer de vestiging waarmee u wilt inloggen" %}</strong></p>
+                {% for branch in company_branches %}
                     {% with company_id=branch.vestigingsnummer %}
-                        <label class="left" for="branch-{{ company_id }}">
-                            <p>{{ branch.naam }}</p>
-                            {% if branch.straatnaam and branch.plaats %}
-                                <p>{{ branch.straatnaam }}, {{ branch.plaats}}</p>
-                            {% endif %}
-                            {% if branch.vestigingsnummer %}
-                                <p>{% trans "Vestiging" %}: {{ company_id }}</p>
-                            {% endif %}
-                        </label>
-                        <input type="radio" id="branch-{{ company_id }}" name="branch_number" value="{{ company_id }}">
+
+                        <li class="choice-list__item">
+                            <label class="choice-list__label">
+                                <input type="radio" id="branch-{{ company_id }}" name="branch_number" value="{{ company_id }}" class="choice-list__radio">
+                                <div class="choice-list__content">
+                                    <h2 class="choice-list__heading">{{ branch.naam }}</h2>
+                                    {% if branch.straatnaam %}
+                                    <p class="choice-list__p">{{ branch.straatnaam }}</p>
+                                    {% endif %}
+                                    {% if branch.plaats %}
+                                    <p class="choice-list__p">{{ branch.plaats}}</p>
+                                    {% endif %}
+                                    {% if branch.vestigingsnummer %}
+                                    <p class="choice-list__p">{% trans "Vestiging" %}: {{ company_id }}</p>
+                                    {% endif %}
+                                </div>
+                            </label>
+                        </li>
+
                     {% endwith %}
-                {% endrender_card %}
-            {% endfor %}
-            {% button_row %}
-                {% button primary=True name="submit" text=_('Log in') %}
-                {% url 'logout' as logout_url %}
-                {% button danger=True href=logout_url text=_('Cancel') %}
-            {% endbutton_row %}
-        </form>
-    {% endrender_column %}
-{% endrender_grid %}
+                {% endfor %}
+
+
+                {# first only show HOOFDVESTIGING #}
```

Comment:
The branch without `vestigingsnummer` is not the `hoofdvestging`, but a pseudo-branch that we create to represent the company as a whole. The `hoofdvestiging` is one vestiging among others, and as such it also has a `vestigingsnummer`.

Reply:
That's right, maybe we should define the "entire company" option outside of `company_branches` to avoid confusion? Or would that make the form more complicated?

Reply:
@pi-sigma  I've left he commented-out code for the split-template still in it, for future use ; 
It's all inside one large unordered-list.
feel free to approve this one and/or make changes in this PR itself.

Reply:
@jiromaykin I could modify it to use the split template, I'll approve and assign the PR to myself :+1: